Library/SharedFunctionsTests.swift

```diff
@@ -346,4 +346,52 @@ final class SharedFunctionsTests: TestCase {
 
     XCTAssertTrue(isEndDateAfterToday(for: reward))
   }
+
+  func testIsAccessTokenEnvelopeEmailVerified_IsEmailVerified_True_FeatureFlag_True() {
```

Comment:
These test names will need to be updated because they're not saying necessarily that the access token email is verified but that email verification should be shown.